lib/src/matchers/jest_dom/css_class_matchers.dart

```diff
@@ -234,16 +233,16 @@ class _ClassNameMatcher extends Matcher {
   @override
   Description describeMismatch(dynamic item, Description mismatchDescription, Map matchState, bool verbose) {
     final missingClasses = matchState['missingClasses'] as Set;
-    final unwantedClasses = matchState['unwantedClasses'] as Set;
-    final extraneousClasses = matchState['extraneousClasses'] as List;
+    final unwantedClasses = matchState['unwantedClasses'] as Set?;
+    final extraneousClasses = matchState['extraneousClasses'] as List?;
 
     final descriptionParts = <String>[];
     if (allowExtraneous) {
-      if (unwantedClasses.isNotEmpty) {
+      if (unwantedClasses!.isNotEmpty) {
```

Comment:
I don't follow how this is safe if we're saying its possibly null on line 236?

Reply:
Good question - I'll add a null check